.gitignore

```diff
@@ -43,3 +43,6 @@ jspm_packages
 
 # Webstorm project metadata
 .idea
+
+# macOS
+.DS_Store
```

Comment:
Could you remove this entry? I would prefer this to be managed by your global git config rather than per repository.

https://coderwall.com/p/vz6ymw/how-to-globally-gitignore-ds_store